content/docs/guides/first_steps.md

```diff
@@ -0,0 +1,185 @@
+
+If we were only interested in the 99th percentile latencies, we could use this
+query to retrieve that information:
+
+```
+prometheus_target_interval_length_seconds{quantile="0.99"}
+```
+
+To count the number of returned time series, you could write:
+
+```
+count(prometheus_target_interval_length_seconds)
+```
+
+For more about the expression language, see the
+[expression language documentation](/docs/querying/basics/).
+
+## Using the graphing interface
+
+To graph expressions, navigate to `http://localhost:9090/graph` and use the "Graph" tab.
+
+For example, enter the following expression to graph the per-second rate of all
+storage chunk operations happening in the self-scraped Prometheus:
+
+```
+rate(prometheus_local_storage_chunk_ops_total[1m])
+```
+
+You can experiment with the graph range parameters and other settings.
+
+## Installing the Node Exporter
+
+Collecting metrics from Prometheus alone isn't a good representation of Prometheus' capabilities. So let's use the Node Exporter to monitor our first resource. We're going to monitor a Linux host called `node.example.com` but you could monitor any Linux or OS X host. There's also [a WMI exporter](https://github.com/martinlindhe/wmi_exporter) for Microsoft Windows hosts too.
+
+[Download the latest release of the Node Exporter](/download/#node_exporter) of Prometheus for your platform, then extract it:
+
+```language-bash
+tar xvfz node_exporter-*.tar.gz
+cd node_exporter-*
+```
+
+The Node Exporter is a single binary, `node_exporter`, and has a configurable set of collectors for gathering various types of host-based metrics. By default, collectors gather [CPU, memory, disk, and other metrics](https://github.com/prometheus/node_exporter#enabled-by-default) and expose them for scraping.
+
+Let's start the node exporter now on our Linux host.
+
+```language-bash
+./node_exporter
+```
+
+The Node Exporter's metrics are available on port `9100` on the host at the `/metrics` path. In our case this is:
+
+`http://node.example.com:9100/metrics`
+
+You can browse to this URL to see the metrics being exposed.
+
+We now need to tell Prometheus about our new target.
+
+## Configuring Prometheus to monitor the host
+
+We will configure Prometheus to scrape this new target. To achieve this, add the following job definition to the `scrape_configs` section in our `prometheus.yml`:
+
+```
+scrape_configs:
+  - job_name: 'prometheus'
+    static_configs:
+      - targets: ['localhost:9090']
+  - job_name:  'node-example'
+    scrape_interval: 5s
+    static_configs:
+      - targets: ['node.example.com:9100']
+        labels:
+          group: 'nodes'
+```
+
+Our new job is called `node-example`. It scrapes a static target, `node.example.com` on port `9100`. You would replace this name with the name or IP address of the host you're monitoring. We also add a label, `group` with a value of `nodes`, to our time series.
+
+Now we restart or `SIGHUP` our Prometheus instance to activate our new job.
```

Comment:
SIGHUP won't work for Windows users, they'll need to activate the lifecycle API and hit the http endpoint. I'd stick with a restart here for simplicitly.